Releases.md

```diff
@@ -0,0 +1,51 @@
+# Release schedule
+
+Current — latest major version. Usually lives for about 6 months, and then becomes Active.
+
+Active — Long-term Support major version which is recommended to have on all projects in the testing pipeline. Active usually lives for about a year, but exceptions are possible. Active will be renewed if necessary to properly test the next Current in order to switch Active to it.
```

Comment:
I'd suggest the lowercase version — `long-term support`. Not critical at all, at your discretion.